torch/_inductor/cuda_properties.py

```diff
@@ -11,15 +13,21 @@
 
 @functools.lru_cache(None)
 def _properties():
-    if not torch.cuda.is_available():
-        return {}
-    try:
-        return {
-            i: torch.cuda.get_device_properties(i)
-            for i in range(torch.cuda.device_count())
-        }
-    except RuntimeError:
-        return {}
+    device_properties = {}
+
+    for _triton_backend in triton_backends:
+        if not _triton_backend:
+            continue
+
+        try:
+            device_properties[_triton_backend.name()] = {
```

Comment:
The properties are for the different Triton backends. Hence, I added the backend name.